File: task-03/prime.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Enter a number ", (num) => {
  for (let i = 2; i <= num; i++) {
    if (isPrime(i)) process.stdout.write(i + " ");
  }
  readline.close();
});

const isPrime = (n) => {
  if (n <= 1) return false;

  for (let i = 2; i < n; i++) if (n % i == 0) return false;

  return true;
};

File: task-03/prime.py
def isPrime(num):
      
    if num <= 1 :
        return False
  
    for i in range(2, num):
        if num % i == 0:
            return False
  
    return True

def main():
    n = int(input("Enter a number: "))

    for i in range(2, n + 1):
        if isPrime(i):
            print(i, end = " ")

if __name__ == "__main__": 
    main()